src/Core/src/Platform/iOS/TransformationExtensions.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using CoreAnimation;
+using CoreGraphics;
+using Microsoft.Maui.Graphics;
+using UIKit;
+
+namespace Microsoft.Maui
+{
+	public static class TransformationExtensions
+	{
+		public static void UpdateTransformation(this UIView nativeView, IView? view)
+		{
+			CALayer? layer = nativeView.Layer;
+			CGPoint? originalAnchor = layer?.AnchorPoint;
+
+			nativeView.UpdateTransformation(view, layer, originalAnchor);
+		}
+
+		public static void UpdateTransformation(this UIView nativeView, IView? view, CALayer? layer, CGPoint? originalAnchor)
```

Comment:
We can't always use layers because some views don't have any. For example text views and steppers and sliders.
We are doing things for the backgrounds for Windows and other things for Android. This PR is "fine" to merge, but not all controls will be transformed - we need that content view.

Reply:
Was a mistake setting the layer in some cases, fixed all the transformations tests.